### DISCUSSION TITLE
confirming new verification email creates account with wrong password

### DISCUSSION THREAD

**Discussion by LORE-D-NATO** [supabase/supabase#21306] [Questions]
When I register a new user under an email, and then do it again using a different password, clicking the second confirmation link creates the verified user with the first password. This means someone can register with my email before me, and then if I register and verify the account, they now have the credentials to access it. Any clues as to why this is happening?

**Comment by GaryAustin1** [comment]
Can you show a sequence of steps with the auth calls your are making?  Also what templates are generating the emails you get?

**Reply by LORE-D-NATO** [reply]
I'm using a new project set up according to this guide: https://supabase.com/docs/guides/auth/server-side/nextjs. Nothing else is changed. Here's a gif of the behavior:

![output](https://github.com/supabase/supabase/assets/115497894/6f1b3c6c-9041-4451-8217-4efd816defd2)
